coalib/processes/Processing.py

```diff
@@ -329,27 +322,32 @@ def instantiate_bears(section,
     :param console_printer:  Object to print messages on the console.
     :return:                 The local and global bear instance lists.
     """
-    local_bear_list = [bear
-                       for bear in filter_raising_callables(
-                           local_bear_list,
-                           RuntimeError,
-                           section,
-                           message_queue,
-                           timeout=0.1,
-                           debug=debug)]
-
-    global_bear_list = [bear
-                        for bear in filter_raising_callables(
-                            global_bear_list,
-                            RuntimeError,
-                            file_dict,
-                            section,
-                            message_queue,
-                            timeout=0.1,
-                            debug=debug)]
-
-    return local_bear_list, global_bear_list
+    instantiated_local_bear_list = []
+    instantiated_global_bear_list = []
+    for bear in local_bear_list:
+        try:
+            debug_flag = debug_flag_checker(bear,debug_bears=debug_bears)
+            instantiated_local_bear_list.append(bear(section,
+                                                     message_queue,
+                                                     timeout=0.1,
+                                                     debug_flag=debug_flag))
```

Comment:
Imo you don't have to switch to a manual-appending-strategy, you can still use the generator-syntax from above^^

Reply:
Nvm